File: javascript/7_kyu/return_a_strings_even_characters.js
// Return a string's even characters
// https://www.codewars.com/kata/566044325f8fddc1c000002c/train/javascript

function evenChars(string) {
    var ans = [];
    
    if (string.length >= 2 && string.length < 100) {
      for (var i = 0; i < string.length; i++) {
        if ( !(i % 2 == 0) ) {
          ans.push(string[i]);
        }
      }
    } else {
      return "invalid string";
    }
    
    return ans;
  }

File: python/7_kyu/return_a_strings_even_characters.py
# Return a string's even characters
# https://www.codewars.com/kata/566044325f8fddc1c000002c/train/python

def even_chars(st): 
    ans = []
    
    if 2 <= len(st) < 100:
        for i in range(0, len(st)):
            if i % 2 != 0:
                ans.append(st[i])
    else:
        return "invalid string"
    
    return ans
                